src/core/createLifecycle.js

```diff
@@ -16,9 +16,6 @@ const hookNames = [
   "onComponentsRegistered",
   // Called before an event is sent on a data collection request
   "onBeforeEvent",
-  // Called before each data collection request
-  // (`interact` or `collect` endpoints)
-  "onBeforeDataCollectionRequest",
```

Comment:
I removed the `onBeforeDataCollectionRequest` hook entirely because we didn't end up using it.